unittests/unit_test.py

```diff
@@ -2,11 +2,6 @@
 import sqlite3 as sql
 import re 
 
-Cu_re=sql.connect('CustomerReservation.db').cursor()
-customer_table=Cu_re.execute('''select * from Customers''').fetchall()
-reservation_table=Cu_re.execute('''select * from Reservation''').fetchall()
-strawberrytype_table=Cu_re.execute('''select * from Strawberrytype''').fetchall()
-
```

Comment:
I moved this code to the if __name__ == '__main__':  statement

Reply:
good